testing/scripts/test_batch_processor.py

```diff
@@ -64,6 +64,42 @@ def test_batch_worker(self, namespace):
 
         logging.info("Finished first batch. Checking.")
 
+        with open(output_data_path, "r") as f:
+            count = 0
+            for line in f:
+                count += 1
+                output = json.loads(line)
+                # Ensure all requests are successful
+                assert output.get("data", {}).get("ndarray", False)
+            assert count == batch_size
+
+        logging.info("Sending first batch (grpc): mini-batch size=1")
+
+        start_multithreaded_batch_worker(
```

Comment:
This function call is getting really ugly imo, way too many arguments and not clear what each one is doing from here. In Go I would suggest we could refactor and pass a struct `BatchWorkerOpts`, but not sure what the most idiomatic Python approach is. (also this is future work not for this PR)